daemon/src/monitor.rs

```diff
@@ -30,7 +30,11 @@ use tokio_tasks::Tasks;
 use xtra_productivity::xtra_productivity;
 use xtras::SendInterval;
 
-const FINALITY_CONFIRMATIONS: u32 = 1;
+const LOCK_FINALITY_CONFIRMATIONS: u32 = 1;
+const CLOSE_FINALITY_CONFIRMATIONS: u32 = 3;
```

Comment:
If you introduce `...Seen` events I think we should be fine. Old CFDs will not have these events, so we should clean them up correctly. I don't see problems with that.

Reply:
> I originally considered introducing a separate event, because I had a similar concern. I was more worried about any business logic that could depend on the {CollaborativeSettlement, Cet, Refund}Confirmed events, but it turns out that only the UI would be affected.

now that you say that. 
I think for `CollabSettlement` it should be fine as we (the maker) are publishing it
For the CET it should also not be a problem because we transition to _closed_ when the oracle attests as well, meaning, we won't miss it neither. 
For refund: we are f**ked anyway and will likely lose some money 😅

Reply:
@da-kami: In order to get this in soon I would opt for accepting this change in UI behaviour for now, record a ticket to introduce the `Seen` events and later on decide whether we want to tackle that or not. Would that be okay?

Reply:
fine with me, we don't have to block this.

Reply:
I opened https://github.com/itchysats/itchysats/issues/1815.